smozROM-alt.py

```diff
@@ -0,0 +1,1004 @@
+#

```

Comment:
I'd like to combine this "alt" version with the regular one, so that it works on both windows and mac and linux. Have you tried it on Windows, or just mac/linux?

Reply:
Oh I didn't even remember that this was a change I made! Ha.. Anyway I'll test it on windows and see if it works.. if it does, we can replace the old one..

Reply:
It did not work on Windows for me, HOWEVER, I work on a Mac and only emulate Windows in dribs and drabs as necessary. I wouldn't trust my ability or inability to get something to work in Windows as proof of anything.